### PR TITLE
beta of Concurrency 3.0

### PR DESCRIPTION
**Pull request by njr-11**
Pull to use for beta of Concurrency 3.0.  This pull was prepared in advance of being ready to save us some time on figuring out what changes are needed to get us through builds when this is eventually ready for beta.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

**Review by tevans78 — CHANGES_REQUESTED** [review]
I think the feature must at least be prioritised before doing this

**Comment by njr-11** [comment]
> I think the feature must at least be prioritised before doing this

I'll be sure not to merge it until that happens.  Prioritization has been waiting on the Upcoming Feature Overview doc, which has been waiting on the specification being far enough along to present one, but it is scheduled for tomorrow (Nov 4) and after that I'll bring this up on the Nov 8 prioritization call.  The goal is to have this ready with the option to beta if it gets approved.  It should be noted that there is quite a lot of matching function that we already get from having our MicroProfile implementation that goes a long way toward being Concurrency 3.0 compliant.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
The build had just one unit test failure in an unrelated area, and is good enough for merging, but this pull will be waiting for other approvals before that happens.

**Comment by njr-11** [comment]
Rebased on to the latest. We still need official copies of the spec api jar and tck before merging.

**Comment by njr-11** [comment]
#run-libby-bot
#build

**Comment by njr-11** [comment]
VisibilityTest found an error, that eeCompatible-10.0 wasn't also marked beta. Fixed this, and rebuilding,
#build
#run-libby-bot

**Comment by njr-11** [comment]
Rebased onto latest to pick up candidate final release concurrency API & TCK
#run-libby-bot
#build

**Comment by njr-11** [comment]
@tevans78 Could you review this now? The feature is prioritized, had its Upcoming Feature Overview presented, has a green build and we're now using the 3.0 candidate for final release.

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-3.0.feature
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-3.0.feature:7]: perhaps doesn't matter so much for a beta but these maven co-ords are wrong.
```suggestion
-bundles=io.openliberty.jakarta.concurrency.3.0; location:="dev/api/spec/,lib/"; mavenCoordinates="io.openliberty.jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:3.0.0.20211206"
```

**Review by tevans78 — APPROVED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-3.0.feature
Comment (njr-11) [reply dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-3.0.feature:7]: I don't see how 20220120 is wrong and the older 20211206 would be correct.  3.0.0.20220120 is how I versioned Liberty's [copy](http://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/io/openliberty/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/3.0.0.20220120/) of the Concurrency spec API jar that is the latest candidate for release of that specification which was built on January 20 (matching the version number).  20211206 is an older development built snapshot driver.  As you point out, by the time this is GA (and probably well before that point), we will be able to switch this to the real location on Maven of the official final release of the spec API jar.

Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-3.0.feature:7]: Yeah, sorry, I looked at the wrong file, my fault

**Pull request merged**
